plutus-core/plutus-ir/src/PlutusIR/Core/Plated.hs

```diff
@@ -115,6 +118,23 @@ bindingIds f = \case
                     <.> PLC.theUnique f n
                     <.*> traverse1Maybe ((PLC.varDeclName . PLC.theUnique) f) vdecls)
 
+-- | Get all the direct constants of the given 'Term' from 'Constant's.
```

Comment:
It just focuses on the contents of the `Constant` constructor (if any) and allows you to either extract it or map it with a function. Actually, looking at it, it's a more general `Prism'` rather than merely a `Traversal'`, but I'm not gonna bother changing it, since a bunch of others need to be changed as well then. We have plenty of those traversals and if they didn't work properly, we'd know that for sure, even just this one would cause tests to fail very quickly, since none of the `isPrintable` logic would work.